client/src/plus/offer-overview/offer-overview-subscribe/stripe.tsx

```diff
@@ -0,0 +1,4 @@
+import "@stripe/stripe-js";
+export default function Stripe() {
+  return <></>;
```

Comment:
I think generally better:
```suggestion
  return null;
```